### ISSUE TITLE
'PeriodIndexResampler' object has no attribute 'index'

### ISSUE THREAD

**Issue by ghost** [ulmo-dev/ulmo#159]
Calling `get_data` triggers an exception:
```pycon
Python 3.6.5 |Anaconda, Inc.| (default, Apr 29 2018, 16:14:56) 
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ulmo
/home/al/.asdf/installs/python/anaconda3-5.2.0/lib/python3.6/site-packages/ulmo/twc/kbdi/core.py:20: FutureWarning: pandas.tslib is deprecated and will be removed in a future version.
You can access Timestamp as pandas.Timestamp
  CSV_SWITCHOVER = pandas.tslib.Timestamp('2016-10-01')
>>> ulmo.ncdc.ghcn_daily.get_data('GM000010147', as_dataframe=True)
/home/al/.asdf/installs/python/anaconda3-5.2.0/lib/python3.6/site-packages/ulmo/ncdc/ghcn_daily/core.py:89: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
  axis=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/al/.asdf/installs/python/anaconda3-5.2.0/lib/python3.6/site-packages/ulmo/ncdc/ghcn_daily/core.py", line 95, in get_data
    daily_index = element_df.resample('D').index.copy()
  File "/home/al/.asdf/installs/python/anaconda3-5.2.0/lib/python3.6/site-packages/pandas/core/resample.py", line 97, in __getattr__
    return object.__getattribute__(self, attr)
AttributeError: 'PeriodIndexResampler' object has no attribute 'index'
>>> ulmo.__version__
'0.8.3dev'
>>> import pandas
>>> pandas.__version__
'0.23.0'
```
I tried with different stations and different data sources and get the same error.

**Comment by johanneshorak** [comment]
Same issue here, calling, for instance,

`ulmo.ncdc.ghcn_daily.get_data('FR000007255')`

results in 

```
AttributeError                            Traceback (most recent call last)
<ipython-input-10-c36f331912f9> in <module>()
----> 1 ulmo.ncdc.ghcn_daily.get_data('FR000007255')

/home/hosch/.local/lib/python3.6/site-packages/ulmo/ncdc/ghcn_daily/core.py in get_data(station_id, elements, update, as_dataframe)
     93         # here we're just using pandas' builtin resample logic to construct a daily
     94         # index for the timespan
---> 95         daily_index = element_df.resample('D').index.copy()
     96 
     97         # XXX: hackish; pandas support for this sort of thing will probably be

/home/hosch/.local/lib/python3.6/site-packages/pandas/core/resample.py in __getattr__(self, attr)
     95             return self[attr]
     96 
---> 97         return object.__getattribute__(self, attr)
     98 
     99     @property

AttributeError: 'PeriodIndexResampler' object has no attribute 'index'
```

I tried to call `get_data` for different station ids, issue remains. I also uninstalled ulmo and installed the **master** and the **fix-ghcn-pandas-issue** branches of ulmo on github, to no avail.
pandas version: pandas (0.23.4)

**Comment by johanneshorak** [comment]
Fixed it on [my ulmo fork](https://github.com/johanneshorak/ulmo), and it's working again for me. I submitted a pull request, not sure if I did that correctly.

**Comment by emiliom** [comment]
Addressed a long time ago by PR #164
